src/stream/ws/handshake.c

```diff
@@ -95,7 +97,7 @@ ssize_t ws_handshake(char* output, size_t output_maxlen, const char* input)
 			"%s%s"
 			"\r\n",
 			response,
-			have_protocols ? "Sec-WebSocket-Protocol: char\r\n" : "",
```

Comment:
This was actually supposed to be "chat", not "char". I wonder if changing to to "chat" would have made a difference.

Reply:
it wouldn't have helped, see below. it would only accept binary or base64.